.github/workflows/build_and_push.yml

```diff
@@ -1,5 +1,4 @@
-name: Build and push images to registry
-
+name: Bild and push trezor-user-env image
```

Comment:
This `bild` is a new slang-word? :D Used here at least two times

Reply:
Thanks, I missed this somehow :laughing: